requirements/testproj.txt

```diff
@@ -7,3 +7,4 @@ djangorestframework-recursive>=0.1.2
 dj-database-url>=0.4.2
 user_agents>=1.1.0
 django-cors-headers
+django-oauth-toolkit>=1.3.0
```

Comment:
Hi, made these changes in the project but still it throws the same error. Could you please help me out

Reply:
See https://github.com/axnsan12/drf-yasg/blob/master/tox.ini#L21 for where else it's pulling the dep from